src/bun.js/bindings/ZigGlobalObject.cpp

```diff
@@ -3325,8 +3334,26 @@ void GlobalObject::installAPIGlobals(JSClassRef* globals, int count, JSC::VM& vm
             JSC::JSObject* dnsObject = JSC::constructEmptyObject(this);
```

Comment:
now that there's a bunch of functions on this guy let's make it a `LazyProperty` so that we skip creating these 10 functions at start